Misc/NEWS.d/next/Library/2019-01-14-14-13-08.bpo-35674.kamWqz.rst

```diff
@@ -0,0 +1 @@
+os.posix_spawnp is now available.
```

Comment:
:func:`os.posix_spawnp` is now available.

Reply:
Add also, `Patch by Joannah Nanjekye` :)